main.go

```diff
@@ -154,7 +157,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
+	err = kongCtx.Run(ctx, client)
+	if err != nil {
+		if k8serrors.IsForbidden(err) && !nctl.Verbose {
+			err = errors.New("Permission denied: are you part of the organization?")
```

Comment:
```suggestion
			err = errors.New("permission denied: are you part of the organization?")
```

This should fix the linter error.